packages/rancher-vsphere/rancher-vsphere-csi/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-version: 100.1.0+up2.3.0
+version: 100.2.0+up2.4.1
```

Comment:
`100.1.1+up2.4.1` should suffice right? Why do we need to bump the minor version?

Reply:
Because this updates 2.3.0 to 2.4.1 and there's been times in the past where bumping the patch for upstream updates caused painful situations for introducing Rancher specific patches. So for vSphere specifically, it is safer if we just bump minor if upstream bumps minor as well.

On a side note, since vSphere charts are local, the +up version represents the 2.5 variant of the same vSphere chart as in `+up<2.5-version>`. Does that mean this should actually be 100.2.0 instead? 2.4.1 does not exist in 2.5, and I don't think there are plans to backport it since 2.5 is EOM. The way we treat the +up for vSphere is confusing because what's going to happen when we have 2.7 charts? Would it be then something like 200.x.x+up100.x.x?